CONTRIBUTING.rst

```diff
@@ -263,27 +158,31 @@ good or fail and where it fails.
 -  If you're having difficulty making unit tests, first consider whether
    your code could be restructured to make it less dependent on other
    modules. Often, however, extra techniques are needed to break down a
-   complex test into simpler ones. @alexcjohnson or @giulioungaretti are
-   happy to help with this. Two ideas that are useful here:
--  Patching, one of the most useful parts of the
-   `unittest.mock <https://docs.python.org/3/library/unittest.mock.html>`__
-   library. This lets you specify exactly how other functions/objects
-   should behave when they're called by the code you are testing.
-
--  Supporting files / data: Lets say you have a test of data acquisition
-   and analysis. You can break that up into an acquisition test and an
-   analysis by saving the intermediate state, namely the data file, in
-   the test directory. Use it to compare to the output of the
-   acquisition test, and as the input for the analysis test.
-
--  We have not yet settled on a framework for testing real hardware.
-   Stay tuned, or post any ideas you have as issues!
+   complex test into simpler ones. We are happy to help with this on Slack.
+   Two ideas that are useful here:
+
+   -  Patching, one of the most useful parts of the
+      `unittest.mock <https://docs.python.org/3/library/unittest.mock.html>`__
+      library. This lets you specify exactly how other functions/objects
+      should behave when they're called by the code you are testing.
+   -  Supporting files / data: Lets say you have a test of data acquisition
+      and analysis. You can break that up into an acquisition test and an
+      analysis by saving the intermediate state, namely the data file, in
+      the test directory. Use it to compare to the output of the
+      acquisition test, and as the input for the analysis test.
+
+-  Refer to QCoDeS documentation on how to implement tests for the 
+   instrument drivers.
+
+   -  We have not yet settled on a framework for testing real hardware.
+      For some tests we use `pyvisa-sim <https://github.com/pyvisa/pyvisa-sim>`__ 
```

Comment:
@jenshnielsen @guenp i added a short notion of pyvisa-sim here, can be improved/expanded in the future. I also refer the readers to qcodes docs, becasue for example pyvisa-sim use is explained there https://qcodes.github.io/Qcodes/examples/writing_drivers/Creating-Simulated-PyVISA-Instruments.html . do you agree with this minimal improvement?

Reply:
Yes I think that is good for now :)